ansible/files/bootstrap/telemetry.sh

```diff
@@ -137,6 +145,28 @@ sudo yum makecache
 # Install packages
 sudo yum -y install git jq htop tmux libffi-devel aws-cli postgresql-devel zsh snappy-devel readline-devel emacs nethogs w3m
 
+# Install agents for metrics and logging
+if [ "$METRICS_PROVIDER" = "stackdriver" ] || [ "$LOGGING_PROVIDER" = "stackdriver" ]; then
+    # Install GCP credentials for stackdriver; see https://cloud.google.com/logging/docs/agent/authorization
+    sudo mkdir -p /etc/google/auth/
+    sudo aws s3 cp $TELEMETRY_CONF_BUCKET/credentials/gcp.json /etc/google/auth/application_default_credentials.json
+fi
+if [ "$LOGGING_PROVIDER" = "stackdriver" ] ; then
+    # https://cloud.google.com/logging/docs/agent/installation
+    sudo bash -c "$(curl -sLS https://dl.google.com/cloudagents/install-logging-agent.sh)"
+fi
+if "$IS_MASTER" ; then
+    if [ "$METRICS_PROVIDER" = "datadog" ] ; then
+        # Install datadog-agent; see https://app.datadoghq.com/account/settings#agent/aws
+        export DD_API_KEY="$(sudo aws s3 cp $TELEMETRY_CONF_BUCKET/credentials/datadog -)"
+        sudo bash -c "$(curl -sLS https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
+    fi
+    if [ "$METRICS_PROVIDER" = "stackdriver" ] ; then
```

Comment:
I don't think we're moving forward with stackdriver for metrics

Reply:
So we're planning to keep Datadog for metrics longer term? Do we have that documented anywhere at this point?

Reply:
I would talk to Brian Pitts about this.  I don't think the long term solution has been decided yet

Reply:
Talked to Brian. Sounds like Datadog for the near or medium term, so I'm going to remove the stackdriver bits from this PR.